wicket-core/src/main/java/org/apache/wicket/page/PartialPageUpdate.java

```diff
@@ -68,6 +69,12 @@
 {
 	private static final Logger LOG = LoggerFactory.getLogger(PartialPageUpdate.class);
 
+	/**
+	 * Meta data to be added to the partial page response.
+	 * This is the response meta data, not HTML meta data.
+	 */
+	protected final List<CharSequence> meta = Generics.newArrayList();
```

Comment:
OK. I see I can call the `addMeta()` method more than once. But still I have no idea when to use this meta items. Some examples in the javadoc might be useful.